src/Middleware/NavigationMiddlewareFactory.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Zend\Navigation\Middleware;
+
+use Interop\Container\ContainerInterface;
+use Zend\Navigation\Navigation;
+
+class NavigationMiddlewareFactory
+{
+    /**
+     * Top-level configuration key indicating navigation configuration
+     *
+     * @var string
```

Comment:
> Please just note it is proposed standard, not yet accepted.

Right, therefore the hint to phpDocumentor. 😉 
As far as I know, APiGen does the same like phpDocumentor.

> It's a bit odd that constant has annotation variable, isn't it?

I do not know where comes from. But, if it is not supported, why should we use that. And if we follow the PSR-5 proposal, than the "type" of the value is described, not the element itself.